### DISCUSSION TITLE
Adding of a new 4 viewport view in the editor in style of CAD or 3D modeling software [Quality of Life]

### DISCUSSION THREAD

**Discussion by TheRealBlackNet** [Redot-Engine/redot-proposals#74] [Ideas]
# Feature
Please add a new "4 viewport (editor sytle)" entry, with the four views prepared in style of a 3D modeling software, like Cinema4D or Blender, to allow a quicker workflow while composing meshes.

# Reason
For composing its often necessary to move objects onto a surfaces or place elements next to each other, three orthogonal views are the best to do this in a 3D enviroment. The current four views by default create only four 3D perspective views the user has to set to match the behavoior of a 3D modeling or CAD software for each scene edited. That costs time and if the user don't repeat it correct the views changes positions and directions that leads to errors like moving elements in the wrong direction because one view showed right view intead of left view on a scene.

# Details
After activating the new option:
View 1 should be the free 3D view, like before to allow easy switching back and forth.
Views 2-4 should be all "view rotation locked" to avoid switching it back into perspective view. 
View 2 front view
View 3 top view
View 4 left view

# Example 
![grafik](https://github.com/user-attachments/assets/e1345c97-53c8-425f-bee3-268ba2a2f5d5)

**Comment by Arctis-Fireblight** [comment]
Seems like a pretty reasonable suggestion.
This is approved if any community member wants to make a PR for this.
